plugins/AvatarHover.plugin.js

```diff
@@ -169,29 +156,23 @@ global.AvatarHover = function () {
     lastTarget = target;
     isShown = settings.isShown || AsyncKeystate.key("Control");
     isLarge = settings.isLarge || AsyncKeystate.key("Shift");
-
     if (!(isShown && target)) {
       return hoverCard.remove();
     }
-
     size = isLarge && 256 || 128;
     boundsTarget = target.getBoundingClientRect();
     boundsWindow = {
       width: window.innerWidth,
       height: window.innerHeight
     };
     left = Math.max(0, boundsTarget.left + (boundsTarget.width - size) / 2);
-
     if (left + size > boundsWindow.width) {
       left = boundsWindow.width - size;
     }
-
     top = size > boundsWindow.height ? (boundsWindow.height - size) / 2 : boundsTarget.bottom + size > boundsWindow.height ? boundsTarget.top - size : boundsTarget.bottom;
-
-    if ("none" === (imageUrl = (((ref = target.querySelector("img")) != null ? ref.src : void 0) || target.src || getComputedStyle(target).backgroundImage.match(/^url\((["']?)(.+)\1\)$/)[2]).replace(/\?size=\d{2,4}\)?$/, `?size=${size}`))) {
+    if ("none" === (imageUrl = (((ref = target.querySelector("img")) != null ? ref.src : void 0) || target.src || getComputedStyle(target).backgroundImage.match(/^url\((["']?)(.+)\1\)$/)[2]).replace(/\?size=\d{3,4}\)?$/, `?size=${size}`))) {
```

Comment:
This change makes it so that some avatars stay at 28px because 28 is a 2 digit number and not 3.

Reply:
this change makes avatars pixelated, you shouldn't do it